src/chapters/booleans-and-conditionals/logical-operators.rst

```diff
@@ -0,0 +1,273 @@
+
+Logical OR
+^^^^^^^^^^
+
+.. index:: ! ||
+
+.. index::
+   single: operator; boolean
+
+JavaScript's logical OR operator, ``||``, also creates compound boolean expressions. This operator takes two operands, and the resulting expression is ``true`` if *either* of the operands are ``true`` individually. If both operands are ``false``, the overall expression is ``false``.
+
+As with logical AND, logical OR mirrors our experience of English language truth values. The sentence "Pigs can fly or dogs can run," is true as a whole. Joining the two clauses by "or" requires that only one of them is true in order for the full sentence to be true. Similarly, "Pigs can fly or dogs can speak Spanish," is a false statement.
+
+Let's look at some examples in JavaScript.
+
+.. sourcecode:: js
+   :linenos:
+
+   console.log(7 > 5 || 5 > 3);
+   console.log(7 > 5 || 2 > 3);
+   console.log(2 > 3 || 'dog' === 'cat');
+
+**Output**
+
+::
+
+   true
+   true
+   false
+
+Lines 1 and 2 both return ``true`` because at least one of the comparison expressions joined by ``||`` is ``true``. Line 3 returns ``false`` because both sub-expressions are ``false``.
+
+.. warning:: The single symbols ``&`` and ``|`` are themselves valid JavaScript operators, so accidentally leaving off one of the symbols off when typing ``&&`` or ``||`` will not result in an error. The resulting value of the expression will be different than expected though.
+
+Logical NOT
+^^^^^^^^^^^
+
+.. index:: ! !
+
+.. index::
+   single: operator; boolean
+
+The logical NOT operator, ``!``, takes only a single operand and reverses its boolean value.
+
+.. admonition:: Example
+
+   .. sourcecode:: js
+
+      console.log(! true);
+      console.log(! false);
+
+   **Output**
+
+   ::
+
+      false
+      true
+
+The operator ``!`` (sometimes called "bang") has the same semantic role as the word "not" in English. 
+
+.. admonition:: Example
+
+   .. sourcecode:: js
+
+      console.log( !(5 > 7) );
+      console.log( !('dog' === 'cat') );
+
+   **Output**
+
+   ::
+
+      true
+      true
+
+Both of the overall expressions in this example return ``true`` because without the ``!`` each is ``false``.
+
+Operator Precedence
+-------------------
+
+.. index::
+   single: operator; precedence
+
+We now have a number of operators in our toolkit. It is important to understand how these operators relate to each other with respect to **operator precedence**. Operator precedence is the set of rules that dictate in which order the operators are applied.
+
+JavaScript will always apply the logical NOT operator, ``!``, first. It then applies the arithmetic operators (``**`` is highest, then ``*`` and ``/``, then ``+`` and ``-``). Next comes the comparison operators. Finally, the logical AND and OR are applied last. 
```

Comment:
Phrasing: "JavaScript always applies the logical NOT operator, ``!``, first. Next, it applies the arithmetic operators, followed by the comparison operators. The logical AND (``&&``) and OR (``||``) are applied last."

There is no need to explain the arithmetic order of operations, since that information is show in the table below.